packages/executor/src/modules/eth.ts

```diff
@@ -190,21 +192,11 @@ export class Eth {
       callGasLimit,
     };
 
-    const gasFee = await getGasFee(
-      this.chainId,
-      this.provider,
-      this.config.etherscanApiKey,
-      {
-        entryPoint,
-        userOp: userOpToEstimate,
-      }
-    );
+    const gasFee = await this.skandhaModule.getGasPrice();
 
     if (this.pvgEstimator) {
-      userOpComplemented.maxFeePerGas =
-        gasFee.maxFeePerGas ?? gasFee.gasPrice ?? 1;
-      userOpComplemented.maxPriorityFeePerGas =
-        gasFee.maxPriorityFeePerGas ?? gasFee.gasPrice ?? 1;
+      userOpComplemented.maxFeePerGas = gasFee.maxFeePerGas;
+      userOpComplemented.maxPriorityFeePerGas = gasFee.maxPriorityFeePerGas;
       preVerificationGas = await this.pvgEstimator(
```

Comment:
Should we consider adding a markup here?

Reply:
it's already added in `skandhaModule.getGasPrice`